worker-languagedetection-container/testcases/automated/CAF_4002/chinese_hmong.yaml.testcase

```diff
@@ -61,35 +61,35 @@ expectedOutputData:
       DetectedLanguage1_ConfidencePercentage:
         action: "replace"
         values:
-        - data: "77"
+        - data: "56"
       DetectedLanguage1_Name:
         action: "replace"
         values:
-        - data: "Chinese"
+        - data: "CHINESE"
       DetectedLanguage2_Code:
         action: "replace"
         values:
-        - data: "hmn"
+        - data: "unknown"
       DetectedLanguage2_ConfidencePercentage:
         action: "replace"
         values:
-        - data: "22"
+        - data: "15"
       DetectedLanguage2_Name:
         action: "replace"
         values:
-        - data: "HMONG"
+        - data: "UNKNOWN"
```

Comment:
Is Hmong no longer supported? If so remove this testcase and update any relevant documentation.

Reply:
Remove this testcase as we are not supporting Hmong